test/tools/yulopti.cpp

```diff
@@ -127,6 +141,19 @@ class YulOpti
 			sortedOptions.push_back(optionAndDescription);
```

Comment:
Should those go to the end? If yes, it should be done after sorting.

Reply:
Depends. Do you want `VarNameCleaner` and `StackCompressor` at the end or grouped with other steps? I thought having them in alphabetical order too was better but I can change it.

The downside of putting them at the end is that the `quit` option ends up after all steps but before `VarNameCleaner` and `StackCompressor` in that case. I'd have to hard-code it as a special case and I really didn't want to do that.

Reply:
ok, then it's fine.